From 05ecfd8ee59e4b333d09db9fe748d659d3342bcd Mon Sep 17 00:00:00 2001
From: Jon West <electrikjesus@gmail.com>
Date: Sat, 11 Mar 2023 12:21:53 -0500
Subject: [PATCH 01/15] Add tablet mode boot options for FFMPEG as well

---
 install/grub2/efi/boot/android.cfg | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/install/grub2/efi/boot/android.cfg b/install/grub2/efi/boot/android.cfg
index 966439e0..7b1371a4 100644
--- a/install/grub2/efi/boot/android.cfg
+++ b/install/grub2/efi/boot/android.cfg
@@ -84,6 +84,8 @@ export android bootefi grub kdir live src
 
 # Create main menu
 add_entry "$live" quiet
+add_entry "$live (Default) w/ FFMPEG" quiet FFMPEG_CODEC=1 FFMPEG_PREFER_C2=1
+add_entry "$live (Intel) w/ FFMPEG" quiet HWC=drm_minigbm_celadon GRALLOC=minigbm FFMPEG_CODEC=1 FFMPEG_PREFER_C2=1
 add_entry "$live PC-Mode (Default)" quiet PC_MODE=1
 add_entry "$live PC-Mode (Default) w/ FFMPEG" quiet PC_MODE=1 FFMPEG_CODEC=1 FFMPEG_PREFER_C2=1 
 add_entry "$live PC-Mode (Intel)" quiet PC_MODE=1 HWC=drm_minigbm_celadon GRALLOC=minigbm
-- 
2.34.1

