From cc6b33867eb272730aa77e6b0e161460d46141f1 Mon Sep 17 00:00:00 2001
From: Jon West <electrikjesus@gmail.com>
Date: Sat, 25 Mar 2023 18:40:10 -0400
Subject: [PATCH 03/15] Add wait for auto_detect

---
 initrd/init | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/initrd/init b/initrd/init
index 302352fd..77903f15 100755
--- a/initrd/init
+++ b/initrd/init
@@ -314,8 +314,10 @@ done
 
 # ensure keyboard driver is loaded
 if [ -n "$INSTALL" -o -n "$DEBUG" ]; then
+	echo "Detecting hardware..."
 	busybox modprobe -a atkbd hid-apple
 	auto_detect &
+	wait
 fi
 
 if [ 0$DEBUG -gt 0 ]; then
-- 
2.34.1

